src/index.css

```diff
@@ -0,0 +1,99 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;1,300;1,400&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  outline: 0;
+  border: 0;
+  text-decoration: none;
+  list-style: none;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  background: #f6f6f6;
+}
+
+.container {
+  width: 50%;
+  margin: 8rem auto 0;
+  padding: 2rem;
+  background: #f9f9f9;
+  box-shadow: rgba(100, 100, 111, 0.2) 0 7px 29px 0;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
```

Comment:
- [ ] Please make sure to add checkboxes to tasks to follow the given design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/235742316-e510799d-a36f-4b14-98ee-5ba4d2860a21.png)

### Given design

![image](https://user-images.githubusercontent.com/83803414/235742249-8d559194-aef4-482e-b6d2-473d8f26a5e6.png)

Reply:
Thanks for the feedback. I'll add it asap.